hazelcast-hibernate/hazelcast-hibernate3/src/test/java/com/hazelcast/hibernate/local/LocalRegionCacheTest.java

```diff
@@ -94,6 +95,16 @@ public void testThreeArgConstructorRegistersTopicListener() {
         verify(topic).addMessageListener(isNotNull(MessageListener.class));
     }
 
+    @Test
+    public void testTimestampToString() {
+        assertNotNull(new Timestamp().toString());
+    }
+
+    @Test
+    public void testInvalidationToString() {
+        assertNotNull(new Invalidation().toString());
+    }
+
     public static void runCleanup(LocalRegionCache cache) {
```

Comment:
As far as I understand, `cleanup` method of `LocalRegionCache` is package-protected and it is needed by a `HazelcastQueryResultsRegionTest` in a different package. It simply delegates the task for a test which cannot access the method.